common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -1430,10 +1431,10 @@
     "figura.docs.texture.get_pixel": "Gets the RGBA color from the specified pixel",
     "figura.docs.texture.set_pixel": "Sets the RGBA color of the specified pixel",
     "figura.docs.texture.fill": "Sets the RGBA color of the entire specified region",
-    "figura.docs.texture.update": "Updates the texture to the gpu, applying all the changes",
+    "figura.docs.texture.update": "Updates the texture to the GPU, applying all the changes",
     "figura.docs.texture.restore": "Restores the texture to its original state, before any modifications",
     "figura.docs.texture.save": "Returns a base64 string representation of this texture",
-    "figura.docs.texture.apply_func": "Calls the given function on the specified area of this texture, it will iterate over each pixel, and their current x, y and color will be given as arguments, the color in a RGBA vec4 format, and the return value will set that pixel color\nInvalid return values or nil takes no effects",
+    "figura.docs.texture.apply_func": "Calls the given function on the specified area of this texture, it will iterate over each pixel, and their current x, y, and color will be given as arguments, the color in an RGBA vec4 format, and the return value will set that pixel color\nInvalid return values or nil takes no effects",
```

This feels slightly more readable.
```suggestion
    "figura.docs.texture.apply_func": "Calls the given function on the specified area of this texture, it will iterate over each pixel, giving its current x, y, and color as arguments, the color is a vec4 in RGBA format, and the return value will set that pixel's color\nInvalid return values or nil takes no effects",
```